opendebates/views.py

```diff
@@ -423,11 +441,11 @@ def report(request, id):
 @allow_http("GET", "POST")
 @login_required
 def merge(request, id):
-    if not allow_voting_and_submitting_questions() and not request.user.is_staff:
+    if not request.site_mode.allow_voting_and_submitting_questions and not request.user.is_staff:
         raise Http404
 
     idea = get_object_or_404(Submission, pk=id)
```

Comment:
`get_object_or_404(Submission, pk=id, site_mode=request.site_mode)`?  Presumably we're not sharing questions between sites.